src/webui/webui.cpp

```diff
@@ -93,7 +93,11 @@ void WebUI::init()
 #endif
 
         if (!m_httpServer->isListening()) {
-            bool success = m_httpServer->listen(QHostAddress::Any, m_port);
+            const QString addressString = pref->getWebUiAddress();
+            const auto address = (addressString == "*" || addressString.isEmpty())
+              ? QHostAddress::Any : QHostAddress(addressString);
```

Comment:
Wrong indentation.

Reply:
I'm assuming you mean the `?`. This was followed from the example given above. Anyway, I've indented it to 4 lines.

Reply:
The examples that we give to each other here, it's not a source for blind copy. We do this often in a hurry. We must take this just as a hint (although I agree, some are pretty finished look).